key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
eek---can you add this to .gitignore and generate a new one? it's in our git commit history forever now :)